collector/incidents.go

```diff
@@ -0,0 +1,86 @@
+package collector
+
+import (
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/digitalocean/godo"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/tidwall/gjson"
+)
+
+const doStatusAPIURL = "https://s2k7tnzlhrpw.statuspage.io/api/v2/summary.json"
+
+// IncidentCollector collects number of active incidents associated with digital ocean services
+type IncidentCollector struct {
+	logger  log.Logger
+	errors  *prometheus.CounterVec
+	client  *godo.Client
+	timeout time.Duration
+
+	Incidents *prometheus.Desc
+}
+
+// NewIncidentCollector returns a new IncidentCollector.
+func NewIncidentCollector(logger log.Logger, errors *prometheus.CounterVec, client *godo.Client, timeout time.Duration) *IncidentCollector {
```

Comment:
No need to pass the `cilent *godo.Client` if it's not used further down.

Reply:
👍